src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,53 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        Manufacturer manufacturerToyota = new Manufacturer(null, "Toyota", "Japan");
```

Comment:
```suggestion
        Manufacturer manufacturerToyota = new Manufacturer("Toyota", "Japan");
```

It is bad practice to pass null to constructor

Reply:
I did it on master by mistake) Will be accurate next time. Thank You Daria. Ok, will change it!